gpkit/tests/t_geometric_program.py

```diff
@@ -31,10 +32,9 @@ def test_trivial_gp(self):
         """
```

Comment:
@bqpd need to think carefully about what this file is really testing. Is it testing `Model` or `GP`? There is no `t_model` currently. Should probably start splitting up tests to match the class boundaries.